src/Core/src/Platform/iOS/TextFieldExtensions.cs

```diff
@@ -32,6 +32,8 @@ public static void UpdateIsPassword(this UITextField textField, IEntry entry)
 			}
 			else
 				textField.SecureTextEntry = entry.IsPassword;
+
+			textField.TextContentType = UITextContentType.OneTimeCode;
```

Comment:
Should use a compilation directive to only apply this change on Catalyst?

Also, It is outside the scope of PR, but it would be interesting to create a Catalyst specific **Platform Specific** allowing to set the https://developer.apple.com/documentation/swiftui/view/textcontenttype(_:)-ufdv (password, emailAddress etc)

Reply:
Since TextFieldExtensions.cs is already platform-specific to iOS, I have confirmed that the mentioned issue is reproducible on both iOS and Mac Catalyst. Therefore, this change will not impact other platforms.